lms/djangoapps/grades/new/course_grade.py

```diff
@@ -65,6 +65,18 @@ def chapter_grades(self):
         }
 
     @lazy
+    def subsection_grades(self):
```

Comment:
Actually, I just created a PR for it: https://github.com/edx/edx-platform/pull/14870